dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/DataAnalysisController.java

```diff
@@ -171,6 +175,17 @@ public class DataAnalysisController {
       throw new WebMessageException(badRequest("No organisation unit defined"));
     }
 
+    int maxResults =
```

Comment:
You can use this for a more readable and compact syntax:

`ObjectUtils.firstNonNull( validationRulesAnalysisParams.getMaxResults(), ValidationService.MAX_INTERACTIVE_ALERTS )`

https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/ObjectUtils.html